go-controller/pkg/metrics/master.go

```diff
@@ -568,3 +607,62 @@ func getPodUIDFromPortBinding(row *sbdb.PortBinding) kapimtypes.UID {
 	}
 	return kapimtypes.UID(podUID)
 }
+
+var metricDBE2eTimestamp = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnNamespace,
+	Subsystem: MetricOvnSubsystemDB,
+	Name:      "e2e_timestamp",
+	Help:      "The current e2e-timestamp value as observed in this instance of the database"},
+	[]string{
+		"db_name",
+	},
+)
+
+func ovnE2eTimeStampUpdater(direction, database string, nbClient libovsdbclient.Client,
+	sbClient libovsdbclient.Client) {
+	var nbGlobal *nbdb.NBGlobal
+	var sbGlobal *sbdb.SBGlobal
+	var err error
+
+	scrapeAndSetE2ETimestampMetrics := func(options map[string]string) {
+		var v string
+		var ok bool
+
+		if v, ok = options["e2e_timestamp"]; !ok {
+			klog.Errorf("Failed to scrape timestamp from global config options")
```

Comment:
klog.Error?

Reply:
Yeah this is generally what I saw done in the rest of the metrics code, error and try again rather than erroring and killing the go routine

Reply:
oh I meant it should be klog.Error not klog.Errorf

Reply:
Oh yep, good catch!

Reply:
fixed